odl/discr/grid.py

```diff
@@ -1246,8 +1248,8 @@ def uniform_sampling_fromintv(intv_prod, num_nodes, nodes_on_bdry=True):
     # gmin = a + (b - a) / (2 * n - 1).
 
     gmin, gmax = [], []
-    for n, beg, end, on_bdry in zip(num_nodes, intv_prod.begin, intv_prod.end,
-                                    nodes_on_bdry):
+    for n, xmin, xmax, on_bdry in zip(shape, intv_prod.min_pt,
```

Comment:
Dunno, I think "x" is somewhat hard-wired with "position" or "location". I think it's at least clearer than "b" and "e". Alternatives?